Examples/Scripts/Python/truth_tracking_telescope.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import os
+from pathlib import Path
+
+import acts
+import acts.examples
+
+from truth_tracking import runTruthTracking
+
+u = acts.UnitConstants
+
+if "__main__" == __name__:
+
+    srcdir = Path(__file__).resolve().parent.parent.parent.parent
+
+    # detector, trackingGeometry, _ = getOpenDataDetector()
+    # detector, trackingGeometry, decorators = acts.examples.GenericDetector.create()
+    detector, trackingGeometry, decorators = acts.examples.TelescopeDetector.create(
+        bounds=[200, 200], positions=[30, 60, 90, 120, 150, 180, 210, 240, 270]
+    )
+
+    field = acts.ConstantBField(acts.Vector3(0, 0, 2 * u.T))
+
+    runTruthTracking(
```

Comment:
So for the truth tracking there is no pre-configured `addTruthTracking`. I guess for a use case like this, reusing `runTruthTracking` makes sense. What do you think @timadye?

Reply:
Sorry, I didn't yet look into the truth tracking, so didn't make an `addTruthTracking`. Maybe I should add that to the list of improvements.
But since `runTruthTracking` does everything needed, maybe we can keep this with a `runTruthTracking` call even when there is the option to use `addTruthTracking`. Or we can switch over.

Reply:
I'll interpret this as "no strong preference" and go ahead with this PR